src/show.js

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
```

Comment:
We might be able to combine these imports like:

`import { Box, Card, Container, Typography } from "@mui/material";`